deploy/synology_dsm.sh

```diff
@@ -79,7 +81,7 @@ synology_dsm_deploy() {
 
   # Login, get the token from JSON and session id from cookie
   _info "Logging into $SYNO_Hostname:$SYNO_Port"
-  response=$(_get "$_base_url/webman/login.cgi?username=$SYNO_Username&passwd=$SYNO_Password&enable_syno_token=yes")
+  response=$(_get "$_base_url/webman/login.cgi?username=$SYNO_Username&passwd=$SYNO_Password&enable_syno_token=yes&device_id=$SYNO_DID")
```

Comment:
how about the existing users, who didn't have `SYNO_DID` saved before?
Can they renew their certs automatically ?

Reply:
@Neilpang yes, an empty value for the URL parameter doesn't cause any problem.